scripts/us_bjs/nps/test_import.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2020 Google LLC
```

Comment:
nit: the general naming convention for a test for <program_name>.py is <program_name>_test.py.

Reply:
renamed